nixos/modules/services/monitoring/prometheus/exporters/ipsec.nix

```diff
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, options }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.exporters.ipsec;
+in
+{
+  port = 9536;
+  serviceOpts = {
+    serviceConfig = {
+      DynamicUser = false;
+      User = "root"; # root user is required to access the ipsec control socket
```

Comment:
are you sure this can't be done with a capability? even if it's `CAP_NET_ADMIN`?